hs/src/Reach/AST/SL.hs

```diff
@@ -828,6 +828,7 @@ data SLPrimitive
   | SLPrim_Contract_new
   | SLPrim_Contract_new_ctor DLContractNews
   | SLPrim_toStringDyn
+  | SLPrim_Bytes_fromHex Integer
```

Comment:
I don't think the Int is necessary because this always results in a static bytes